nx-X11/programs/Xserver/hw/nxagent/Screen.c

```diff
@@ -171,6 +171,9 @@ WindowPtr nxagentViewportFrameBelow;
 Bool nxagentCreateScreenResources(ScreenPtr pScreen);
 void nxagentPrintAgentGeometry(char *hdrMessage, char *prefix);
 
+int nxagentAdjustRandRXinerama(ScreenPtr pScreen);
```

Comment:
This should be moved to `nx-X11/programs/Xserver/hw/nxagent/Screen.h` between `nxagentChangeScreenConfig` and `nxagentSaveAreas`.